codeland.html

```diff
@@ -645,24 +729,31 @@
             systems. She also co-hosts the Base.cs Podcast, and is a producer of the
             BaseCS and Byte Sized video series.
           </p>
-          <button class="crayons-btn crayons-btn--secondary" type="button" data-url="">
-            See discussion
-          </button>
+          <div class="codeland-livestream__speaker-info__footer">
+            <button class="crayons-btn crayons-btn--secondary w-100" type="button" data-url="">
+              See discussion
+            </button>
+          </div>
         </div>
       </aside>
-      <div class="codeland-livestream__info">
-        <span class="ff-accent fs-base">
-          Currently
-        </span>
-        <div class="codeland-livestream__info-title">
-          <h2>
-            The Cost of Data by Vaidehi Joshi
-          </h2>
-          <h2 class="codeland-livestream__info-helper fw-normal">
-            View Info
-          </h2>
+      <a href="" class="codeland-livestream__info-link">
```

Comment:
Not sure if it's been discussed already, I know @joshpuetz is working on schedule stuff, but the doubt just came to mind: Whether or not we'll have the discussion articles published and linked from here before each talk airs.

Reply:
I do believe we are planning on having those articles are written up and published ahead of time!